runtime/lua/vim/lsp.lua

```diff
@@ -1053,7 +1053,9 @@ end
 --- already for this client, then force-shutdown is attempted.
 ---
 ---@param client_id integer|integer[]|vim.lsp.Client[] id, list of id's, or list of |vim.lsp.Client| objects
----@param force? boolean shutdown forcefully
+---@param force? boolean|integer Whether to shutdown forcefully.
+--- If `force` is a number, it will be treated as the time in milliseconds to
+--- wait before forcing the shutdown.
 function lsp.stop_client(client_id, force)
```

Comment:
@justinmk @lewis6991 thoughts on deprecating this function? Feels like yet another wrapper of something that could simply be `vim.lsp.get_client_by_id(id):stop(force)`.